src/Styleguide/Components/FollowArtistPopover/index.tsx

```diff
@@ -1,30 +1,72 @@
-import { Flex } from "@artsy/palette"
+import { Box, color, Flex, Sans, space } from "@artsy/palette"
 import { FollowArtistPopover_suggested } from "__generated__/FollowArtistPopover_suggested.graphql"
 import { FollowArtistPopoverQuery } from "__generated__/FollowArtistPopoverQuery.graphql"
 import { ContextConsumer, ContextProps } from "Artsy/SystemContext"
+import Icon from "Components/Icon"
 import React, { SFC } from "react"
 import { createFragmentContainer, graphql, QueryRenderer } from "react-relay"
+import styled from "styled-components"
 import { FollowArtistPopoverRowFragmentContainer as FollowArtistPopoverRow } from "./FollowArtistPopoverRow"
 
+// TODO: Revisit and style this better.
+const Container = styled.div`
+  background-color: white;
+  border-left: 1px solid black;
+  border-right: 1px solid black;
+  border-bottom: 1px solid black;
+`
+
+const TitleContainer = styled.div`
+  background-color: white;
+  border-left: 1px solid black;
+  border-right: 1px solid black;
+  border-top: 1px solid black;
+`
+
+const CloseIcon = styled(Icon)`
+  color: ${color("black30")};
+  cursor: pointer;
+  font-size: 12px;
+`
+
 interface Props extends ContextProps {
   suggested: FollowArtistPopover_suggested
 }
 
+interface TitleProps {
+  onClose?: () => void
+}
+
+export const FollowArtistPopoverTitle: SFC<TitleProps> = props => {
+  return (
+    <TitleContainer>
+      <Sans size="1" weight="medium">
+        Other artists you might like
+      </Sans>
+      <Box position="absolute" top={space(1)} right={space(1)}>
```

Comment:
to be able to position the cursor in the top right of the popover/modal

Reply:
You don't need them -- our components go through palette. 

Is the above screencap correct? Its seems like the arrow should be directly in the middle of the button rather than floating to the right in space.

Reply:
The arrow is actually in the center of a 'follow artist button container', which is the width of that entire sidebar. And acutally if you click on the empty space on the right hand side, you'll trigger the follow. So I think the popover is positioning itself properly, there's just something off with that button/container (or maybe the popover should move into the container and just wrap the icon/button).